Example_Systems/RealSystemExample/ISONE_Singlezone/Settings/highs_settings.yml

```diff
@@ -7,318 +7,9 @@ Pre_Solve: choose          # Presolve option: "off", "choose" or "on" # [type: s
-mip_max_leaves: 2147483647
-
-# limit on the number of improving solutions found to stop the MIP solver prematurely
-# [type: HighsInt, advanced: false, range: {1, 2147483647}, default: 2147483647]
-mip_max_improving_sols: 2147483647
-
-# maximal age of dynamic LP rows before they are removed from the LP relaxation
-# [type: HighsInt, advanced: false, range: {0, 32767}, default: 10]
-mip_lp_age_limit: 10
-
-# maximal age of rows in the cutpool before they are deleted
-# [type: HighsInt, advanced: false, range: {0, 1000}, default: 30]
-mip_pool_age_limit: 30
-
-# soft limit on the number of rows in the cutpool for dynamic age adjustment
-# [type: HighsInt, advanced: false, range: {1, 2147483647}, default: 10000]
-mip_pool_soft_limit: 10000
-
-# minimal number of observations before pseudo costs are considered reliable
-# [type: HighsInt, advanced: false, range: {0, 2147483647}, default: 8]
-mip_pscost_minreliable: 8
-
-# minimal number of entries in the cliquetable before neighborhood queries of the conflict graph use parallel processing
-# [type: HighsInt, advanced: false, range: {0, 2147483647}, default: 100000]
-mip_min_cliquetable_entries_for_parallelism: 100000
-
-# MIP solver reporting level
-# [type: HighsInt, advanced: false, range: {0, 2}, default: 1]
-mip_report_level: 1
-
-# MIP feasibility tolerance
-# [type: double, advanced: false, range: [1e-10, inf], default: 1e-06]
-mip_feasibility_tolerance: 1e-06
-
-# effort spent for MIP heuristics
-# [type: double, advanced: false, range: [0, 1], default: 0.05]
-mip_heuristic_effort: 0.05
-
-# tolerance on relative gap, |ub-lb|/|ub|, to determine whether optimality has been reached for a MIP instance
-# [type: double, advanced: false, range: [0, inf], default: 0.0001]
-mip_rel_gap: 0.0001
-
-# tolerance on absolute gap of MIP, |ub-lb|, to determine whether optimality has been reached for a MIP instance
-# [type: double, advanced: false, range: [0, inf], default: 1e-06]
-mip_abs_gap: 1e-06
-
-# Output development messages: 0 => none; 1 => info; 2 => verbose
-# [type: HighsInt, advanced: true, range: {0, 3}, default: 0]
-log_dev_level: 0
 
 # Run the crossover routine for IPX
 # [type: bool, advanced: true, range: {false, true}, default: true]
-run_crossover: false #true
-
-# Allow ModelStatus::kUnboundedOrInfeasible
-# [type: bool, advanced: true, range: {false, true}, default: false]
-allow_unbounded_or_infeasible: false
-
-# Use relaxed implied bounds from presolve
-# [type: bool, advanced: true, range: {false, true}, default: false]
-use_implied_bounds_from_presolve: false
-
-# Prevents LP presolve steps for which postsolve cannot maintain a basis
-# [type: bool, advanced: true, range: {false, true}, default: true]
-lp_presolve_requires_basis_postsolve: true
-
-# Use the free format MPS file reader
-# [type: bool, advanced: true, range: {false, true}, default: true]
-mps_parser_type_free: true
+run_crossover: true #true
```

Comment:
Why is this true here? and why is true commented out?

Reply:
For the ISONE_Singlezone case, I needed to turn ON the crossover, otherwise the IPM iterations prematurely ends and no result is output. I will put a note against it. But, haven't yet been able to find out why.